modules/geolocation/unittests.py

```diff
@@ -202,6 +202,100 @@ def test_point_set_2(self):
         actual = self.locator.calculate_pixel_to_geo_mapping()
         # Test
         np.testing.assert_almost_equal(actual, expected)
+        
+        
+class TestOutputAnalysis(unittest.TestCase):
+    """
+    Tests Geolocation.outputanalysis(inputLocationTupleList)
+
+    """
+    def test_all_ones(self):
+        outputAnalysis = geolocation.OutputAnalysis()
+
+    # Setup
+        outputList = np.array([[[1, 1], [1], [1]]])
```

Comment:
To give any future developers a better idea of what's happening here, it would be better to define what expectedInputLocationTupleList first, and then define expectedCoordPair and expectedError from that

Reply:
Got it, so a change in variable name from outputList to expectedInputLocationTupleList?

Reply:
No, maybe something like `listToAnalyze`, since it isn't an actual output from the function being tested.